nsv13/modules/credits/credits.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+	<style>
+		body {
+			overflow: hidden;
+			background-color: black;
+			color: white;
+			font-family: Georgia, serif
+		}
+		#container {
+			position: relative;
+			width: 100vw;
+			text-align: center;
+		}
+		#splash {
+			display: table-cell;
+			height: 100vh;
+		}
+		.crewtable{
+			width: 100%;
+			table-layout: fixed;
+			white-space: nowrap;
+		}
+		.crewtable td{
+			white-space: nowrap;
+			overflow: hidden;
+			text-overflow: ellipsis;
+			font-size: 1.17em;
+		}
+		.actorname {
+			text-align: right;
+			float: right;
+		}
+		.actorsegue {
+			text-align: center;
+			width: 40px;
+		}
+		.actorrole {
+			float: left;
+			text-align: left;
+		}
+		.letterboxer {
+			width: 10%;
+		}
+		.disclaimers {
+			font-family: Arial, Helvetica, sans-serif
+		}
+		h1, h2, h3 {
+			display: inline;
+			font-weight:normal;
+		}
+
+
+	</style>
+</head>
+<body>
+	<div id="sounddiv">
+		<!-- the soundplayer gets added here via javascript! !-->
+	</div>
+	<table id="container"> <!-- Yes we use a table to center ourselves. StackOverflow says this is not acceptable practice past 2012 but we're operating with 2009 tech so... !-->
+		<tr>
+			<td class="letterboxer"></td>
+			<td id="splash">
+				IF YOU CAN SEE THIS, TELL A CODER WHAT HAPPENED!
```

Comment:
will probably change this to direct them to send a issue report at the top right on this line